Zotero/Scenes/Master/Collections/Models/CollectionsState.swift

```diff
@@ -46,6 +46,8 @@ struct CollectionsState: ViewModelState {
     var trashItemsCountToken: NotificationToken?
     var trashCollectionsCountToken: NotificationToken?
     var itemsChangesToken: NotificationToken?
+    var cachedTrashItemsCount: Int
```

Comment:
Do we need these new vars in both `CollectionsState` and `CollectionsActionHandler`?